src/h5web/metadata-viewer/EntityTable.tsx

```diff
@@ -37,14 +43,20 @@ function EntityTable(props: Props): ReactElement {
         </tr>
         {(isDataset(entity) || isDatatype(entity)) && (
           <tr>
-            <th scope="row">Type</th>
+            <th scope="row">HDF5 type</th>
             <td>
               {typeof entity.type === 'string'
                 ? entity.type
                 : entity.type.class}
             </td>
           </tr>
         )}
+        {isDataset(entity) && hasBaseType(entity) && (
+          <tr>
+            <th scope="row">Data type</th>
+            <td>{renderBaseType(entity.type)}</td>
+          </tr>
+        )}
```

Comment:
It feels a bit redundant to show the main type twice (e.g. Integer). Don't you think a single _Type_ row would suffice?

Reply:
This will make the condition for displaying the row more complex but I would agree, yes